jellyfin_kodi/helper/api.py

```diff
@@ -206,6 +206,11 @@ def get_file_path(self, path=None):
         if not path:
             return ""
```

Comment:
Yes, but if the path is not supplied then there's nothing to substitute and the file would not be able to be played in native mode either since there's no path for kodi to search in.